scripts/artifactory.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+./gradlew :library:prepareArtifacts
```

Comment:
I can't explain it, but this script is the only way I can get things to work. Gradle task dependencies randomly skips some of these.